libs/wingsdk/src/target-sim/function.inflight.ts

```diff
@@ -40,24 +58,24 @@ export class Function implements IFunctionClient, ISimulatorResourceInstance {
     return this.context.withTrace({
       message: `Invoke (payload=${JSON.stringify(payload)}).`,
       activity: async () => {
-        const vm = new NodeVM({
-          console: "redirect", // we hijack `console.xxx` in `cloud/function.ts`
-          require: {
-            external: true,
-            builtin: ["*"], // allow using all node modules
-            context: "sandbox", // require inside the sandbox (addresses #1871)
-          },
-          sandbox: {
-            $simulator: this.context,
-          },
-          env: {
-            ...process.env,
-            ...this.env,
+        if (!this.bundle) {
+          throw new Error("unable to find bundle (function not initialized?)");
```

Comment:
Would it make sense to check the budled path actually exists rather than just checking the instance variable is set? Incase there was some wonky-sauce with the bundle during init()?